### PR TITLE
Add agenda items for 2020-10-20 plenum

### PR DESCRIPTION
**Pull request by valentine**

### CODE REVIEW

File: contents/plenum/2020-10-20/_index/main.md [hunk 1]
```diff
@@ -44,6 +53,7 @@
 
 ### Memberships
 - (Bob) Update on new members (if any).
+- (Valentine) Request for latest membership breakdown by tiers.
```
Comment (valentine) [inline contents/plenum/2020-10-20/_index/main.md:56]: @hyperair For your action.

File: contents/plenum/2020-10-20/_index/main.md [hunk 2]
```diff
@@ -23,6 +23,8 @@
 4. On an accrual basis we are up/down $?/month ?
     - **including predicted upcoming costs amortised monthly, we are _down_ $620 a month**
 
+- (Valentine) request for expense breakdown by category, on an annualised (365/12) basis.
```
Comment (valentine) [inline contents/plenum/2020-10-20/_index/main.md:26]: @dyanawu For your action.

Comment (dyanawu) [reply contents/plenum/2020-10-20/_index/main.md:26]: Regular Expenses | Amount | Frequency | Monthly Cost (amortised)
-- | -- | -- | --
Rental | 3400 | 12 | $ 3,400.00
Utilities | 450 | 12 | $ 450.00
Internet | 148 | 12 | $ 148.00
CorpSec | 240 | 2 | $ 40.00
Accounting | 3000 | 1 | $ 250.00
Aircon | 980 | 1 | $ 81.67
Cleaning | 80 | 26 | $ 173.34
Income Tax | 600 | 1 | $ 50.00
  |   |   | $ 4,593.01

File: contents/plenum/2020-10-20/_index/main.md (hunk 1 above)
Comment (dyanawu) [reply contents/plenum/2020-10-20/_index/main.md:56]: Tier | 16 | 32 | 64 | 128 | 256 | 512
-- | -- | -- | -- | -- | -- | --
Count | 4 | 0 | 13 | 12 | 1 | 3

2 of the `512`'s will probably drop by December.

File: contents/plenum/2020-10-20/_index/main.md [hunk 3]
```diff
@@ -23,6 +23,8 @@
 4. On an accrual basis we are up/down $?/month ?
     - **including predicted upcoming costs amortised monthly, we are _down_ $620 a month**
```
Comment (dyanawu) [inline contents/plenum/2020-10-20/_index/main.md:24]:
```suggestion
    - **including predicted upcoming costs amortised monthly, we are _down_ $680 a month**
```

Typo

File: contents/plenum/2020-10-20/_index/main.md (hunk 1 above)
Comment (dyanawu) [inline contents/plenum/2020-10-20/_index/main.md:56]:
```suggestion
- (Valentine) Request for latest membership breakdown by tiers.

Tier | 16 | 32 | 64 | 128 | 256 | 512
-- | -- | -- | -- | -- | -- | --
Count | 4 | 0 | 13 | 12 | 1 | 3

dwu: two of the 512s will probably drop by December
```

**Comment by dyanawu** [comment]
All updates above accurate as of 18-10-2020.

File: contents/plenum/2020-10-20/_index/main.md [hunk 4]
```diff
@@ -36,6 +38,13 @@
 ### Organisation Shutdown
 - (Bob, Robert, Dyana) discussion and proposed timeline
 
+### Membership Survey
+- (Valentine) send members a short email asking:
+    1. what their personal vision for a hackerspace is;
+    2. what they hope to get out of such a place;
+    2. why they continue to pay their memberships.
```
Comment (dyanawu) [inline contents/plenum/2020-10-20/_index/main.md:45]:
```suggestion
    1. what their personal vision for a hackerspace is;
    2. what they hope to get out of such a place;
    3. why they continue to pay their memberships.
```

**Pull request merged**